shinysdr/i/top.py

```diff
@@ -392,33 +392,33 @@ def set_source_name(self, value):
             raise ValueError('Source %r does not exist' % (value,))
         self.source_name = value
         self._do_connect()
-    
+
     @exported_value(type=NoticeT(always_visible=False), changes='continuous')
     def get_clip_warning(self):
         level = self.__clip_probe.level()
         # We assume that our sample source's absolute limits on I and Q values are the range -1.0 to 1.0. This is a square region; therefore the magnitude observed can be up to sqrt(2) = 1.414 above this, allowing us some opportunity to measure the amount of excess, and also to detect clipping even if the device doesn't produce exactly +-1.0 valus.
-        if level >= 1.0:
+        if level >= 1.0 and level <= 1.414: # math.sqrt(level):
```

Comment:
The comment "math.sqrt(level):" is too cryptic. Is it perhaps leftover commented out code?

(I can also see an improvement here: if the level is _ever_ above sqrt(2) then disable this warning. That will help with a device whose level temporarily drops into the warning range. I think I'll do that myself if you don't care to.)